15/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/configuration.md

```diff
@@ -1,71 +1,46 @@
-# Rich Text Editor Configuration
+# Settings
 
-## Configuration
+In this article you can learn about the different options you have for configuring the Rich Text Editor (RTE).
 
-{% embed url="<https://www.youtube.com/watch?v=QRIWz9SotY4>" %}
-Rich Text Editor default implementation
-{% endembed %}
-
-In this article you can learn about the various ways you can configure the RTE.
-
-### Toolbar
+## Toolbar
 
 You have full control over which options should be available on the RTE.
 
-![Toolbar: All options enabled](../../built-in-property-editors/rich-text-editor/images/toolbar-full-11.png)
+![Toolbar: All options enabled](images/rte-tiptap-all-toolbar-items.png)
 
-In the examble above, all 34 options have been enabled. The options include copy/paste buttons, font styles like bold and italics, bullet lists and options to embed videos and insert images.
+In the example above, all 27 options have been enabled. These options include font styles like bold and italics, bullet lists, and options to embed videos and insert images.
 
-### Stylesheets
+You can customize the look of the toolbar on the Data Type using the Rich Text Editor property editor.
 
-It is possible to define specific styles that can be used when editing content using the RTE. You can use as many of these styles with the RTE as you want.
+![Enhance and customize the capabilities of the Rich Text Editor toolbar](images/rte-tiptap-capabilities-and-toolbar.png)
 
-The RTE styles are defined in CSS files which can be created in the **Settings** section. Read the [RTE Styles](rte-styles.md) article to learn more about this feature.
+* Enhance the capabilities of the toolbar by enabling or disabling extensions.
```

Comment:
This list seems out of place cz it's right below an image without a leading line.